README.rst

```diff
@@ -74,23 +74,57 @@ for a development install::
 
 install for tinas
 *****************
-for windows peeps you can start by getting `conda installed`_
-and the `C++ build toolz`_ on your system.
+for windows peeps you can start by installing all the prerequisite software:
 
-then, `crack a conda shell`_ and run the following commands::
+- install git with all default settings - https://git-scm.com/download/win
```

Comment:
so i'd recommend doing a hyperlink syntax here instead of the explicit urls, though if you think that's a little 🤯 for tinas then this is fine too.

basically you'd do something like

```markdown
- [install git with all default settings](https://git-scm.com/download/win)
- [install anaconda all default settings](https://www.anaconda.com/products/individual)
...
etc.
etc.
```

Reply:
I personally like seeing the url so I know what I am clicking on.